.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: aEX812GNCA5bquIsnnnhsRvjIR8tPpb8
```

Comment:
If this is added to .gitignore, shouldn't it be removed from git index?

Reply:
The reason for having the test and models file in the PR is that the initial project had the article model file missing, so we had to create it for it to pass
Initially, we commented it when setting up travis but we decided it was not a good idea so we had to set it up

Reply:
what about https://github.com/andela/annie-hall-cfh/pull/7#discussion_r152777861?

Reply:
yes, it was a mistake on our part, we should have to delete it on github and include it in our .gitignore